factory/usr/lib/systemd/system/classic-mounts.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=Create mount units specific to classic
+OnFailure=emergency.target
+OnFailureJobMode=replace-irreversibly
+
+DefaultDependencies=no
+
+Requires=initrd-root-fs.target
+After=initrd-root-fs.target
```

Comment:
I think it needs `Before=initrd-fs.target`. If somehow `initrd-fs.target` is reached before the daemon-reload, I am not totally sure the generated mounts will be started.

Reply:
Good point, I've added this